### ISSUE TITLE
FR: Settings for 

### ISSUE THREAD

**Issue by mgonzales71** [irzyxa/Volume2#139]
**Describe the solution you'd like**

**Comment by irzyxa** [comment]
Already exists in left top corner of settings screen, near Move, Close menu.

**Comment by mgonzales71** [comment]
yes but i mean where you can allow the selection of osd and tray skins for light mode and dark modes

**Comment by mgonzales71** [comment]
maybe i need to explore this setting a bit more - see if it affects the skins
issue i am having is skin that uses black text is not visible when os in dark mode and white test same when in light mode *(tray skin specifically)

**Comment by irzyxa** [comment]
Can you attach some screenshots?

**Comment by mgonzales71** [comment]
When in light mode this skins text is not visible so would option to have a skin that is used when in Dark Mode and another skin to use in light mode.

figured that would be easier then having skins that had a dark and light mode version built into skin

![JIEixDXBKk](https://user-images.githubusercontent.com/6510026/108405470-f65a0700-71d5-11eb-8894-27b3bc1dd130.png)

does that make sense?

**Comment by mgonzales71** [comment]
so would be able to set skin "volume2 default dark" when system in light mode and "volume2 default light" when in dark mode

**Comment by irzyxa** [comment]
Do you often change mode?

**Comment by mgonzales71** [comment]
i use application https://github.com/Armin2208/Windows-Auto-Night-Mode to change mode at sunrise and sunset
so yes

**Comment by irzyxa** [comment]
I only thought about this :)

**Comment by mgonzales71** [comment]
:-) well just a feature request and obviously not huge deal
maybe can find out a way to check windows current mode and use a config.ini file for that mode then run volume2.exe /lc if needed

**Comment by irzyxa** [comment]
I think I can add a ew tab to Interface settings for System schema - Tray Icon and user can select one for Dark and another for Light mode.
![image](https://user-images.githubusercontent.com/5119360/108407028-db19e680-722b-11eb-834f-6f2b4fdb4d2c.png)

**Comment by mgonzales71** [comment]
that would work - maybe also for OSD skin too?
thanks!

**Comment by mgonzales71** [comment]
while you are considering this you may want to change the "system" interface setting to follow light / dark mode colors

http://www.mediafire.com/file/3ukeryzw41qpfz8/ClassicShellUtility.exe/file

is a cool utility to read all the current theme's interface colors (it's dynamic)

link is from http://www.classicshell.net/downloads/

![image](https://user-images.githubusercontent.com/6510026/108951979-be135800-761d-11eb-96fb-fe6fb4ae506c.png)

might want to consider it so you can have the default interface option be "auto" and follow the system theme colors and support dark and light mode in your app.

i am not really windows programmer but maybe this and that app can help

https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.application.requestedtheme?view=winrt-19041

that app was used by skins for classicshell

so you could set skin elements like such as an example

```
Main_background=$SystemAccentDark1|$StartBackground
Main_font="Segoe UI",normal,-9
Main_text_color=$StartPrimaryText,$StartSelectionPrimaryText,$StartSecondaryText,$StartSelectionSecondaryText
Main_selection=$SystemAccentDark2|$StartSelectionBackground
Caption_font="Segoe UI",normal,18
Caption_text_color=$StartPrimaryText
Main_separator=$SystemAccentDark1|$StartBackground
Main_separator_tint1=$StartSystemTilesBackground
```

in fact the source code is here and may help with all that color stuff 
could end up being able to support new volume2 skins with support for something similar

https://github.com/Open-Shell/Open-Shell-Menu

again no rush just something to think about when bored hehe

thanks!

**Comment by mgonzales71** [comment]
a simplified UI color previewer 
https://www.microsoft.com/store/productId/9NWKG085308X

looks like it shows the most common UI element variant names / colors 

![image](https://user-images.githubusercontent.com/6510026/108954527-c7062880-7621-11eb-9dfe-0e609925cb24.png)

**Comment by o000u** [comment]
> 15 Dec 2021

agree!!👍🏻👍🏻

**Comment by o000u** [comment]
> 

agree!!👍🏻👍🏻

**Comment by o000u** [comment]
> that would work - maybe also for OSD skin too? thanks!

agree!!👍🏻👍🏻

**Comment by irzyxa** [comment]
So some changes here, Interface settings was moved to main screen
![image](https://user-images.githubusercontent.com/5119360/151210244-a64e7cae-14ee-4170-939c-e1521e7abe40.png)

**Comment by irzyxa** [comment]
@iamMG The test version is available:
https://drive.google.com/file/d/1dNIEHjWOZ98WPkJvjKB6ZStyU-t6GIbs/view?usp=sharing
![image](https://user-images.githubusercontent.com/5119360/232225821-1c9d5009-f73c-419a-823d-4b7208c2ea10.png)

**Comment by iamMG** [comment]
Thanks. I will try it out.
I still don't understand when the **Alternate Tray Skin** comes into play.

**Comment by irzyxa** [comment]
You have to select Alternative device
![image](https://user-images.githubusercontent.com/5119360/232226983-b873c214-eb67-44cf-b44c-e78b423f0117.png)

**Comment by iamMG** [comment]
Aah I see. Query resolved finally. Thanks.
